kiliautoml/models/_hugging_face_named_entity_recognition_model.py

```diff
@@ -249,6 +165,73 @@ def tokenize_and_align_labels(examples):
         trainer.save_model(ensure_dir(path_model))
         return output.training_loss
 
+    def predict(
+        self,
+        *,
+        assets: List[AssetT],
+        model_path: Optional[str],
+        from_project: Optional[str],
+        batch_size: int,
```

Comment:
not completely sure that we should let the user control the batch size for prediction.

Reply:
is there a reason why we should?

Reply:
it's necessary for the tests on GitHub action. there is not enough ram. And it's cool to enable the user this parameter in case he possesses a lot of unused ram